multisig-wallet/project/contracts/test-artifacts/callable-contract/src/main.sw

```diff
@@ -0,0 +1,47 @@
+contract;
+
+use std::context::msg_amount;
+
+abi CallableContract {
+    #[storage(write)]
+    fn change_mapping_without_value(address: Address, value: u64);
+
+    #[payable]
+    #[storage(write)]
+    fn change_mapping_with_value(address: Address, value: u64);
+
+    #[storage(read)]
+    fn check_counter_map(address: Address) -> u64;
+
+    #[storage(read)]
+    fn check_deposit_map(address: Address) -> u64;
+}
+
+storage {
+    counter_map: StorageMap<Address, u64> = StorageMap {},
+    deposit_map: StorageMap<Address, u64> = StorageMap {},
+}
+
+impl CallableContract for Contract {
+    #[storage(write)]
+    fn change_mapping_without_value(address: Address, value: u64) {
```

Comment:
```suggestion
    fn update_counter(address: Address, value: u64) {
```

Reply:
Although I'd normally agree with your suggestions for these function names; as this is a test artifact having the overly-explicit `change_mapping_without_value` makes it much clearer in the tests whats supposed to be happening when calling these functions.

Reply:
No. There's already an equivalent amount of prior information that you would need to use your verbose name.

`change` or `update`, `update` is canonical.
`mapping`, which one? Is there more than one? What does the mapping do?
`without_value` implies there is one with a value but what is the value? Payable, parameter?

If anything, someone who may not know what the contract does `update_counter` conveys more information than changing some mapping with some value.

Reply:
Fair point, will implement your suggetsions